src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java

```diff
@@ -179,6 +179,34 @@ public void testGetLine() throws Exception {
                 .isEqualTo(" * I'm a javadoc");
     }
 
+    @Test
+    public void testGetLineCodePoints() throws Exception {
+        final AbstractCheck check = new AbstractCheck() {
+            @Override
+            public int[] getDefaultTokens() {
+                return CommonUtil.EMPTY_INT_ARRAY;
+            }
+
+            @Override
+            public int[] getAcceptableTokens() {
+                return getDefaultTokens();
+            }
+
+            @Override
+            public int[] getRequiredTokens() {
+                return getDefaultTokens();
+            }
+        };
+        check.setFileContents(new FileContents(new FileText(
+                new File(getPath("InputAbstractCheckTestFileContents.java")),
+                Charset.defaultCharset().name())));
+        final int[] expectedCodePoints = {32, 32, 32, 32, 112, 117, 98, 108, 105, 99, 32, 105, 110,
```

Comment:
Is there a reason that we can't do `"somestring".codePoints().toArray();` here?

Reply:
done

Reply:
Resolved.